twine/commands/upload.py

```diff
@@ -91,6 +92,34 @@ def _make_package(
     return package
 
 
+def _split_inputs(
+    inputs: List[str],
+) -> Tuple[List[str], Dict[str, str], Dict[str, List[str]]]:
```

Comment:
I almost wonder if at this point it would be better to use a Namedtuple to give greater significance to the fields etc.

Reply:
Sounds good! I think that'll improve readability a lot here.

Reply:
Done! I've added an `upload.Inputs(NamedTuple)` that encapsulates the current return values.